Recrutify/Recrutify.DataAccess/Repositories/CandidateRepository.cs

```diff
@@ -143,6 +143,16 @@ public Task UpdateStatusByIdsAsync(IEnumerable<Guid> ids, Guid projectId, Status
             return GetCollection().UpdateManyAsync(filter, updateDefinition, updateOptions);
         }
 
+        public Task UpdateIsAssignedAsync(IEnumerable<InterviewAppointmentSlot> interviewAppointmentSlots, Guid projectId)
+        {
+            var updateBuilder = Builders<Candidate>.Update;
+            var updateModels = interviewAppointmentSlots.Select(i => new UpdateOneModel<Candidate>(
```

Comment:
Rewrite it adequately, so that someone except you can read it. Don't use multiple ternary operations, it's not readable at all. And get rid of duplications, you have difference only in value which you set, not in Set method with field which you update

Reply:
the second parameter in the Set() method has a TField, i cannot insert the ternary operator there

Reply:
you shouldn't use ternary operations in repository at all!

Reply:
![image](https://user-images.githubusercontent.com/47914279/144615595-52ed4e1e-bfa2-466e-9f5a-15f13f0a7cba.png)